schema.sql

```diff
@@ -37,3 +37,25 @@ ALTER TABLE animals
 ADD CONSTRAINT FK_species
 FOREIGN KEY (species_id)
 REFERENCES species (id);
+
+
+DROP TABLE IF EXISTS vets;
```

Comment:
- [ ] (Optional) If you want to drop a table you can the `CASCADE` keyword. The `CASCADE` keyword will drop a table and all dependencies, that is a table with all foreign constraints. 👍 Presently, if you try to drop it, it will give you an error if the table has a reference to foreign keys. 👍